src/Generator.php

```diff
@@ -98,35 +162,39 @@ private function allocateLocaleArray($path)
                 if (pathinfo($fileName, PATHINFO_EXTENSION) !== 'php') {
                     continue;
                 }
+
+                $root = realpath(base_path() . config('vue-i18n-generator.langPath') . '/' . $lastLocale);
```

Comment:
since generateMultiple() is conditionally called earlier, and generateMultiple() initializes availableLocales, the  $lastLocale will be empty here if generateMultiple() wasn't called.

I think that is what is causing travis to fail at the moment.